.github/workflows/images.yml

```diff
@@ -4,8 +4,6 @@ on:
     branches:
```

Comment:
Will we be pushing images to DockerHub in case of release/develop ? Shouldn't this be GHCR instead ?